tools/ci_build/build.py

```diff
@@ -576,9 +572,6 @@ def install_ubuntu_deps(args):
             if args.enable_pybind:
                 install_apt_package("python3")
```

Comment:
kind of odd to install python3 when this script is a python script? 
maybe we can get rid of install_ubuntu_deps() and install_apt_package() entirely now.